spec/rubocop/cop/rspec/single_argument_message_chain_spec.rb

```diff
@@ -47,6 +47,39 @@
         end
       RUBY
     end
+
+    context 'with single-key hash argument' do
+      it 'reports an offence' do
+        expect_violation(<<-RUBY)
+          before do
+            allow(foo).to receive_message_chain(bar: 42)
+                          ^^^^^^^^^^^^^^^^^^^^^ Use `receive` instead of calling `receive_message_chain` with a single argument.
+          end
+        RUBY
+      end
+
+      include_examples(
+        'autocorrect',
+        'before { allow(foo).to receive_message_chain(bar: 42) }',
+        'before { allow(foo).to receive(:bar) { 42 } }'
+      )
+
+      include_examples(
+        'autocorrect',
+        'before { allow(foo).to receive_message_chain("bar" => 42) }',
+        'before { allow(foo).to receive(:bar) { 42 } }'
```

Comment:
If so, it would actually make https://github.com/backus/rubocop-rspec/pull/375/files#diff-91336a4d8c49f0937c8d89c6ea457d1cR58 simpler by just being `#to_s`, i think.